js/src/relay/WalletLinkRelay.ts

```diff
@@ -840,4 +848,29 @@ export class WalletLinkRelay implements WalletLinkRelayAbstract {
       }
     })
   }
+
+  public async makeEthereumJSONRPCRequest(
+    request: JSONRPCRequest,
+    jsonRpcUrl: string
```

Comment:
This still requires that the jsonRpcUrl is set.  I think its preferable to make the jsonRpcUrl optional.  Let's make this call fail if it's not provided.

Reply:
Good callout. I left throwing the error inside the provider. It makes more sense to have it here.

Reply:
(3301:5214:205ce2997062)

Reply:
Sorry I mean can we make jsonRpcUrl optional.

Like
```
public async makeEthereumJSONRPCRequest(
    request: JSONRPCRequest,
    jsonRpcUrl?: string
)
```

Reply:
Also can we add a return type for this?

Reply:
I just pushed the change for that!

Reply:
of course! let me add the return type